MultiSlider.js

```diff
@@ -61,6 +64,8 @@ export default class MultiSlider extends React.Component {
       slipDisplacement: 200,
     },
     customMarker: DefaultMarker,
+    markerOffsetX: 0,
```

Comment:
wouldn't it be better if you sat this to default being 24 instead and let the props being overriden when passed instead? Then you would avoid the substraction below? What do you think?